code/modules/clothing/suits/armor/plate_carrier.dm

```diff
@@ -6,15 +6,23 @@
 	color = COLOR_GRAY40
 	valid_accessory_slots = list(ACCESSORY_SLOT_INSIGNIA, ACCESSORY_SLOT_ARMOR_C, ACCESSORY_SLOT_ARMOR_A, ACCESSORY_SLOT_ARMOR_L, ACCESSORY_SLOT_ARMOR_S, ACCESSORY_SLOT_ARMOR_M)
 	restricted_accessory_slots = list(ACCESSORY_SLOT_INSIGNIA, ACCESSORY_SLOT_ARMOR_C, ACCESSORY_SLOT_ARMOR_A, ACCESSORY_SLOT_ARMOR_L, ACCESSORY_SLOT_ARMOR_S)
+	material = /decl/material/solid/leather
+	starting_accessories = null
 
-/obj/item/clothing/suit/armor/pcarrier/light
+/obj/item/clothing/suit/armor/pcarrier/prepared
 	starting_accessories = list(/obj/item/clothing/accessory/armorplate)
 
-/obj/item/clothing/suit/armor/pcarrier/light/press
+/obj/item/clothing/suit/armor/pcarrier/press
 	color = COLOR_BABY_BLUE
+	starting_accessories = list(/obj/item/clothing/accessory/armor/tag/press)
+
+/obj/item/clothing/suit/armor/pcarrier/press/prepared
 	starting_accessories = list(/obj/item/clothing/accessory/armorplate, /obj/item/clothing/accessory/armor/tag/press)
 
 /obj/item/clothing/suit/armor/pcarrier/medium
```

Comment:
this is a pretty confusing path now, because it's not ACTUALLY medium, it's same base pcarrier but with bags. IMO they way these all should be set up:
`/obj/item/clothing/suit/armor/pcarrier` - buildable
`/obj/item/clothing/suit/armor/pcarrier/press` - buildable (because different color
`/obj/item/clothing/suit/armor/pcarrier/light` - not buildable, default plate, for mapping etc
`/obj/item/clothing/suit/armor/pcarrier/medium` - not buildable, medium plate and pouches, for mapping etc
Accessories should be printed separately anyway, because AFAIK cost calculations don't factor them so e.g. /obj/item/clothing/suit/armor/pcarrier and /obj/item/clothing/suit/armor/pcarrier/medium would cost same.